pkg/machinery/client/client.go

```diff
@@ -1007,16 +997,22 @@ func ReadStream(stream MachineStream) (io.ReadCloser, <-chan error, error) {
 			}
 
 			if data.Metadata != nil && data.Metadata.Error != "" {
+				var err error
+
 				if data.Metadata.Status != nil {
-					errCh <- status.FromProto(data.Metadata.Status).Err()
+					err = status.FromProto(data.Metadata.Status).Err()
```

Comment:
I feel we might want to ignore here same errors as above, basically making it `CloseWithError(nil)` is it's Canceled/DeadlineExceeded

Reply:
Done.